### PR TITLE
Secure FO from JS injection in URL

### PR DESCRIPTION
**Pull request by hibatallahAouadni**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.6.1.x
| Description?  | secure token from js injection by htmlentities
| Type?         | bug fix
| Category?     | FO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | http://forge.prestashop.com/browse/PSCSX-9032
| How to test?  | Install the blockwishlist module. Then, access to this URL: http://your-prestashop-path/index.php?token=anytoken%27;alert(%27XSS%27);%27&fc=module&module=blockwishlist&controller=view

### CODE REVIEW

**Comment by hibatallahAouadni** [comment]
@LittleBigDev this one too 1.6.1.17 :)

**Review by maximebiloe — APPROVED** [review]

**Comment by marionf** [comment]
Hello @hibatallahAouadni 

It's not working for me, the alert is still there

pr8171

**Comment by hibatallahAouadni** [comment]
Hello @marionf 
did you apply the modifications that I made in the PR? 
because I test it and the pop-up is not displayed.

File: themes/default-bootstrap/global.tpl [hunk 1]
```diff
@@ -27,7 +27,7 @@
 {addJsDef baseDir=$content_dir}
 {addJsDef baseUri=$base_uri}
 {addJsDef static_token=$static_token}
-{addJsDef token=$token}
+{addJsDef token=$token|htmlentities:$smarty.const.ENT_QUOTES}
```
Comment (hibatallahAouadni) [inline themes/default-bootstrap/global.tpl:30]: ping @marionf ;)

**Comment by marionf** [comment]
It's ok @hibatallahAouadni, it was cache.
Thank you !

**Comment by maximebiloe** [comment]
Thanks @hibatallahAouadni

**Pull request merged**